spec/util/NodeEventEmitterDataType-spec.ts

```diff
@@ -0,0 +1,115 @@
+import { expect } from 'chai';
+
+import {
+  NodeEventEmitterNameDataPair,
+  NodeEventEmitterDataType,
+  NodeEventEmitterDataTypeUnknown,
+  NamedNodeEventEmitter,
+  AnyToUnknown,
+} from '../../src/internal/util/NodeEventEmitterDataType';
+
+describe('NodeEventEmitterDataType smoke testing', () => {
```

Comment:
Same as my other comment. These should be `dtslint` tests. :)

Reply:
Moved to `dtslint`